lib/api/controllers/securityController.js

```diff
@@ -422,18 +440,16 @@ class SecurityController {
    * @returns {Promise<Object>}
    */
   createRestrictedUser(request) {
-    let
-      user = new User(),
-      pojoUser;
-
     assertHasBody(request);
     assertBodyHasNotAttribute(request, 'profileIds');
     assertIdStartsNotUnderscore(request);
 
-    pojoUser = request.input.body;
+    const pojoUser = request.input.body;
     pojoUser._id = request.input.resource._id || uuid.v4();
     pojoUser.profileIds = this.kuzzle.config.security.restrictedProfileIds;
 
+    const user = new User();
+
     return this.kuzzle.repositories.user.hydrate(user, pojoUser)
       .then(modifiedUser => this.kuzzle.repositories.user.persist(modifiedUser, {database: {method: 'create'}}))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 112 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)